ocf_datapipes/load/nwp/providers/gfs.py

```diff
@@ -1,50 +1,50 @@
 """Open GFS Forecast data"""
 
 import logging
-from pathlib import Path
-from typing import Union
 
-import pandas as pd
 import xarray as xr
 
+from ocf_datapipes.load.nwp.providers.utils import open_zarr_paths
+
+# import pandas as pd
```

Comment:
remove commented out code if not needed

Reply:
yeah that's for the commented-out extrapolation bit. it's a bit awkward-looking with all the other imports but idk where else to put it